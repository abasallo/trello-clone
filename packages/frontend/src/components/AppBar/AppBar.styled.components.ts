import styled from 'styled-components'

import MaterialUIAppBar from '@material-ui/core/AppBar'
import MaterialUIToolbar from '@material-ui/core/Toolbar'

import { customMUITheme as theme } from '../../customMUITheme'

export const MaterialUIStyledAppBar = styled(MaterialUIAppBar)`
  background-color: #ffffff;
  z-index: ${theme.zIndex.drawer + 1};
  margin-bottom: 40px;
`

export const Toolbar = styled(MaterialUIToolbar)`
  width: 100%;
  display: flex;
`

export const Logo = styled.img`
  max-width: 192px;
  width: 100%;
  height: auto;
  padding: 15px;
`

export const HeaderLoginButtons = styled.div`
  margin-left: auto;
  text-align: right;
`
