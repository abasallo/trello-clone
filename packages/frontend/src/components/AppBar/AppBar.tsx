import React from 'react'

import logo from '../../images/logo.png'

import constants from '../../modules/constants'

import { Logo, MaterialUIStyledAppBar, Toolbar } from './AppBar.styled.components'

const AppBar = () => (
  <MaterialUIStyledAppBar position="static">
    <Toolbar>
      <Logo src={logo} alt={constants.LOGO_ALT} />
    </Toolbar>
  </MaterialUIStyledAppBar>
)

export default AppBar
