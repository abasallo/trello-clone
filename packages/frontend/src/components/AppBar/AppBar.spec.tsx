import React from 'react'
import renderer from 'react-test-renderer'

import AppBar from './AppBar'

test('Renders properly', () => expect(renderer
    .create(<AppBar user={{email: 'host@server.tld'}}/>).toJSON()).toMatchSnapshot())
