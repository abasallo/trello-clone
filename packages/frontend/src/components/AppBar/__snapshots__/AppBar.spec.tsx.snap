// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders properly 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary sc-bcXHqe eaoBhD MuiPaper-elevation4"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular sc-gswNZR cYoTxS MuiToolbar-gutters"
  >
    <img
      alt="Logo"
      className="sc-dkrFOg iWcAaZ"
      src="logo.png"
    />
  </div>
</header>
`;
